src/config/xorg/index.md

```diff
@@ -1,57 +1,50 @@
 # Xorg
 
-## Drivers
-
-## Display managers
-
-## Session and seat management
+This section details the manual installation and configuration of the Xorg
+display server and common related services and utilities. If you would just like
+to install a full desktop environment, it is recommended to try one of the
+[flavor images](../../installation/live-images/index.md#flavor-images)
 
-Session and seat management is not necessary for every setup, it is used to
-provide device access on the fly for the currently active user session.
+## Installation
 
-For desktop environments like Gnome [elogind](#elogind) is necessary.
+Void provides a comprehensive `xorg` package which installs the server and all
+of the free video drivers, fonts, and base applications. This package is a safe
+option, and should be adequate for most systems which don't require proprietary
+video drivers.
 
-### ConsoleKit2
+If you would like to select only the packages you need, the `xorg-minimal`
+package contains the base xorg server *only*. Remember that you will likely need
+to install a font package (like `xorg-fonts`), a terminal emulator (like
+`xterm`), and a window manager to have a usable graphics system.
 
-Install ConsoleKit2 and activate its service and make sure the dbus and the
-cgmanager services are activated too.
-
-```
-# xbps-install -S ConsoleKit2
-# ln -s /etc/sv/dbus /var/service/
-# ln -s /etc/sv/cgmanager /var/service/
-# ln -s /etc/sv/consolekit /var/service/
-```
+## Drivers
```

Comment:
I'd add a short sentence (or two, if more needs to be said) here to lead into the sub-headings. Something like, "Void provides both open-source and proprietary (non-free) drivers."